cli/ballerina-tool/src/main/java/org/ballerinalang/tool/Main.java

```diff
@@ -68,6 +68,9 @@ public static void main(String... args) {
         } catch (BLauncherException e) {
             LauncherUtils.printLauncherException(e, errStream);
             Runtime.getRuntime().exit(1);
+        } catch (BLangPanicException e) {
+            errStream.println(e.getMessage());
```

Comment:
I think the e.getMessage will not have any useful info as always print "Execution exited with the exit code <some non 0 value>". Runtime already print the actual cause when panic. Therefore we do not need to print the error.  Also this will miss the exit code when we print the echo $? which will be always 1. So we can save the exitValue in PanicException and set in here without printing any thing.

catch (BLangPanicException e) {
Runtime.getRuntime().exit(((BlangPanicException).getExitValue())
}